ci/script.sh

```diff
@@ -4,6 +4,11 @@ set -ex
 
 # TODO This is the "test phase", tweak it as you see fit
 main() {
+    # Add a cfg spec to allow disabling specific tests under CI.
+    if [ ! -z $CI ]; then
+        export RUSTFLAGS=--cfg=trust_ci
```

Comment:
I think it may be better to append this flag to RUSTFLAGS here in case the user may want to set RUSTFLAGS in the env.global field.